metadata.json

```diff
@@ -14,6 +14,13 @@
         "12.04",
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
```

Comment:
Hi all,
I'm not interested in this PR any more, so please feel free to take it over.

On 24 March 2015 at 19:55, Florent Poinsaut <notifications@github.com>
wrote:

> In metadata.json
> <https://github.com/echoes-tech/puppet-monit/pull/4#discussion_r27010659>:
>
> > @@ -14,6 +14,13 @@
> >          "12.04",
> >          "14.04"
> >        ]
> > +    },
> > +    {
> > +      "operatingsystem": "RedHat",
> > +      "operatingsystemrelease": [
>
> As you can see here :
> https://github.com/puppetlabs/puppetlabs-ntp/blob/master/metadata.json
> You can keep the 'operatingsystemrelease' keyword but you have to remove
> '.0' in this versions
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/echoes-tech/puppet-monit/pull/4/files#r27010659>.
>